features/step_definitions/ssp_steps.rb

```diff
@@ -101,15 +101,14 @@
   find(:xpath, "//a[contains(@href, '/#{store.current_listing}')]").click
 end
 
-And /^I check '(.*)' for '(.*)'$/ do |label,field_name|
+And /^I '(.*)' '(.*)' for '(.*)'$/ do |action,label,field_name|
   within "##{field_name}" do
-    check label
-  end
-end
-
-And /^I uncheck '(.*)' for '(.*)'$/ do |label,field_name|
-  within "##{field_name}" do
-    uncheck label
+    case action
+    when "check"
+      check label
+    when "uncheck"
+      uncheck label
+    end
```

Comment:
I think you should include defaults for your case statements.  If someone was to write `I 'go to' 'Wales' for 'locations'`, this test still runs green without complaining.